src/sbt-test/skip-sbt1.4/scala-3/build.sbt

```diff
@@ -0,0 +1 @@
+scalaVersion := "3.3.1"
```

Comment:
`sbt-1.5/*` was ignored on JDK21 until now. The fix was simply to get a compatible scala 3 version (per the [compatibility matrix](https://docs.scala-lang.org/overviews/jdk-compatibility/overview.html)).